src/components/textarea/textarea.component.tsx

```diff
@@ -328,6 +332,7 @@ export const Textarea = React.forwardRef(
           as="textarea"
           inputRef={inputRef}
           validationIconId={validationRedesignOptIn ? undefined : validationId}
+          inputBorderRadius={borderRadius}
```

Comment:
**question** (non-blocking): I was puzzled that we need to set a `borderRadius` on both `InputPresentation` and on `Input` itself. Experimenting in devtools, I see that we do need it on the wrapping div (that's the `InputPresentation`) - but removing it from the `textarea` element itself (which is what `Input` renders here) seems to have no adverse (or other) effect, the corners are still rounded.

I may have missed something having only played with this for 10 seconds, but if I haven't, we can not only get rid of this prop being passsed here, but all the code you've added/changed in the `Input` component to make this work, keeping only the changes to `InputPresentation`.

Reply:
It's needed for the `Input` component too, I believe. If you remove the border radius from the `Input` it looks this screenshot. 

![Screenshot 2024-02-09 at 09 36 43](https://github.com/Sage/carbon/assets/56251247/a74515e9-4450-4a03-b801-a086daf57139)

Reply:
fair enough, let's keep it then. I did say I'd probably missed something (I hadn't actually tried typing anything in 😅 )